### ISSUE TITLE
[Feature Request] Allow option to configure table prefix

### ISSUE THREAD

**Issue by akhilputhiry** [jupyterhub/jupyterhub#2323]
I am trying to install jupyter hub in one of my existing database. Database already has a table called `users`. If you can provide an option to specify `prefix` it would be very helpful.

@yuvipanda

**Comment by minrk** [comment]
Makes sense. Might be a bit tricky, but if there's a general option to sqlalchemy to add a table prefix, that would be useful.
